kafka_connect_healthcheck/parser.py

```diff
@@ -57,4 +57,10 @@ def get_parser():
                         dest="log_level",
                         nargs="?",
                         help="The level of logs to be shown. Default: INFO.")
+
+    parser.add_argument("--basic-auth",
+                        default=os.environ.get("HEALTHCHECK_BASIC_AUTH", "").upper(),
```

Comment:
Hi, thanks for your contribution!

Is there a reason you're doing a `.upper()` on the basic auth?

Reply:
This was just a copy/paste mistake, sorry. Thanks for noticing!